### DISCUSSION TITLE
Autofill hints for IText views

### DISCUSSION THREAD

**Discussion by coen22** [CommunityToolkit/Maui#374] [New Feature Discussions]
Hi everyone,

I'd like to discuss a proposal for autofill in MCT. For that I've already opened an issue and demonstrated some working code from Xamarin Forms, which also works in MAUI. https://github.com/CommunityToolkit/Maui/issues/373

As @brminnick suggested it would be better to use behaviours (instead of the current effect implementation).
So I think it would be a good idea to implement similarly to [TintColorBehavior](https://github.com/pictos/MauiSpecRepo/tree/master/MauiSpecRepo/Behavior)
The `AutofilBehaviour` could extend from `Behaviour<IText>`.

If you have any suggestions, please let me know!

**Comment by bijington** [comment]
@coen22 thanks for this idea, it sounds great. I wonder did you manage to answer your question on whether there is any support for this on Windows? We aim to make sure that features added will be functional on all platforms before we proceed.

**Reply by coen22** [reply]
As far as I can find, this is currently only a feature supported in Edge.

**Comment by TheCodeTraveler** [comment]
Hey @coen22! Have you had a try at redoing the API Proposal for `IText` + `Behavior`? 

If you paste it here we can iterate on it a bit before pushing it to the [Proposal](https://github.com/CommunityToolkit/Maui/issues/373).

**Reply by coen22** [reply]
I'm writing a new version of the proposal and I'll post it here once it's done.

**Comment by coen22** [comment]
I've implemented a first draft of the behaviour. I tried to use the same code style, but the shared class doesn't have '.shared' in it. Also, I need to document a bit, but it's pretty straightforward. Would love to hear what you think. [Archive.zip](https://github.com/CommunityToolkit/Maui/files/8823457/Archive.zip)

**Reply by coen22** [reply]
@brminnick it seems to work, however not within a Shell. Not all fields are filled, but it's up to the autofill provider. On iOS and macOS some things need to be fixed.

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889

**Reply by fredericDelaporte** [reply]
I am unable to find anything related to the discussion here from this YouTube link. Is it possible to have an abstract of the closure reason in plain text here please?
